src/App.tsx

```
@@ -17,6 +17,7 @@ function App() {
   const [messages, setMessages] = useState<Message[]>([]);
   const [isStreaming, setIsStreaming] = useState(false);
   const [preference, setPreference] = useState<"movie" | "anime" | null>(null);
+  const name='ankit'
```

ESLint

[error] 'name' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)